### PR TITLE
Add __setitem__ for json array objects

### PR DESCRIPTION
**Pull request by TylerSheffels**
This resolves  http://manage.dimagi.com/default.asp?161566#895533

### CODE REVIEW

**Comment by TylerSheffels** [comment]
@dannyroberts

**Comment by dannyroberts** [comment]
Awesome looks great. Good to go after tests

**Comment by dannyroberts** [comment]
@TylerSheffels let me know if you need help getting this into pypi (I suspect you will)

**Comment by czue** [comment]
fyi: https://confluence.dimagi.com/display/commcarehq/Python+Packaging+Crash+Course

File: jsonobject/containers.py [hunk 1]
```diff
@@ -31,6 +31,10 @@ def __delitem__(self, i):
         super(JsonArray, self).__delitem__(i)
         del self._obj[i]
 
+    def __setitem__(self, index, wrapped):
+        self._obj[index] = wrapped
```
Comment (dannyroberts) [inline jsonobject/containers.py]: @TylerSheffels This caused a bug, fixed https://github.com/dimagi/jsonobject/pull/122. Internal representations need to be unwrapped.

**Pull request merged**